content/docs/try-react.md

```diff
@@ -0,0 +1,39 @@
+---
+id: try-react
+title: Try React
+permalink: docs/try-react.html
+next: add-react-to-a-new-app.html
+redirect_from:
+  - "docs/installation.html"
+  - "download.html"
+  - "downloads.html"
+  - "docs/tooling-integration.html"
+  - "docs/package-management.html"
+  - "docs/language-tooling.html"
+  - "docs/environments.html"
+---
+
+Try React online or set up your local development environment.
+
+## Online
+
+If you're just interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s).
+
+## HTML Template
+
+If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so don't use it in production.
+
+## Next Steps
+
+### Quick Start
+
+- Head over to the [Quick Start](/docs/hello-world.html) section for a step-by-step introduction to React concepts.
+
+- Try the [Tutorial](/tutorial/tutorial.html) for a hands-on practical example.
+
+### Install React
```

Comment:
Again, this sounds like it's a required step because of the title.

How about: "Powerful Development Environment" or "Complete Development Environment". Maybe also change the sentence to explain when/why you'd want that.

The goal is to explain this step takes time but gives you a great developer experience and a production-ready setup with linting, testing, and optimizations built-in. We have a chance to preempt "omg why is it 100mb" knee-jerk responses here by being explicit that it's not just for 'hello world' but more IDE-like.

Reply:
👍